openedx/core/djangoapps/schedules/content_highlights.py

```diff
@@ -14,6 +14,24 @@
 log = logging.getLogger(__name__)
 
 
+def get_all_course_highlights(course_key):
+    """
+    Does the course have any highlights for any section/week in it?
+    This ignores access checks, since highlights may be lurking in currently
+    inaccessible content.
```

Comment:
Was this logic confirmed? Just want to make sure we want to be pushing up highlights to potentially inaccessible content. Could see this having a perhaps undesired effect if send highlights to a learner about what they'll learn and then they go to the course and can't do it.

Although it might be desired to be like, check back next week when you're going to learn about ... 

So just want to confirm we are getting what we want here.

Reply:
will double check

Reply:
asked Kaitlin here https://edx-internal.slack.com/archives/G019J2RMNEA/p1614886801015800 but i'm pretty sure this is the same logic as the dynamic pacing emails based on looking at the code? though i'm not actually super familiar with how those work